packages/url_launcher/url_launcher_android/pigeons/messages.dart

```diff
@@ -33,9 +41,14 @@ abstract class UrlLauncherApi {
   /// Opens the URL externally, returning true if successful.
   bool launchUrl(String url, Map<String, String> headers);
 
-  /// Opens the URL in an in-app Custom Tab or WebView, returning true if it
-  /// opens successfully.
-  bool openUrlInApp(String url, bool allowCustomTab, WebViewOptions options);
+  /// Opens the URL in an in-app WebView, returning true if it opens
```

Comment:
Why was "Custom Tab" removed from this comment?

Reply:
Cannot quite recall. Looking back I had no reason to do that. Reverted